ru/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -322,127 +322,127 @@
+<!ENTITY pbQuoteConversationButton.tooltiptext "Конспект беседы">
 <!ENTITY pbToggleReplyQuotingMode.accesskey   "u">
 <!ENTITY pbToggleReplyQuotingMode.key   "O">
 <!ENTITY pbFocusIdentityPicker.key "R">
 
 <!-- Insert Link Dialog -->
 
-<!ENTITY pbInsertLink.label "Enter the Internet address (URL) for this link.">
+<!ENTITY pbInsertLink.label "Введите адрес в Интернете (URL-адрес) для этой ссылки.">
 <!ENTITY pbInsertLink.accesskey "E">
 
-<!ENTITY pbInsertImage.label "Enter a Internet address (URL) or local file location:">
+<!ENTITY pbInsertImage.label "Введите адрес в Интернете (URL-адрес) или местонахождение файла:">
 <!ENTITY pbInsertImage.accesskey "E">
 
-<!ENTITY insertListButton.tooltip "Apply or remove bulleted or numbered list">
+<!ENTITY insertListButton.tooltip "Применить или отменить маркированный или нумерованный список">
 
 
 <!--
  entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY sendAndArchive.label "Send and Archive">
-<!ENTITY sendLaterAndArchive.label "Send Later and Archive">
+<!ENTITY sendAndArchive.label "Отправить и архивировать">
+<!ENTITY sendLaterAndArchive.label "Отправить позже и архивировать">
 
 
 <!--
  entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY newResponse.label "New Response...">
-<!ENTITY editResponses.label "Edit Responses...">
-<!ENTITY responseButton.tooltip "Insert a response">
-<!ENTITY responseButton.label "Responses">
+<!ENTITY newResponse.label "Новый ответ…">
+<!ENTITY editResponses.label "Изменить ответы…">
+<!ENTITY responseButton.tooltip "Вставить ответ">
+<!ENTITY responseButton.label "Ответы">
 
 
 <!--
 entities below were automatically added by l10nmerge
 -->
 
 <!ENTITY pbTopicContextMenu-assignColor.accesskey   "c">
-<!ENTITY pbTopicContextMenu-delete.label       "Delete">
+<!ENTITY pbTopicContextMenu-delete.label       "Удалить">
 <!ENTITY pbTopicContextMenu-delete.accesskey   "d">
-<!ENTITY pbTopicContextMenu-assignColor.label  "Set Topic Color">
+<!ENTITY pbTopicContextMenu-assignColor.label  "Установить цвет метки">
 
 
 <!--
 entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY pbStatsWords.label "words:">
+<!ENTITY pbStatsWords.label "слов:">
 <!ENTITY pbSourceView.accesskey "d">
 <!ENTITY pbSourceViewThemes.accesskey "T">
-<!ENTITY pbStatsChars.label "characters:">
-<!ENTITY formattingToolbar.label "Toolbar">
-<!ENTITY pbShowCompositionGoalsToolbarCmd.label "Goal Status Bar">
-<!ENTITY pbStatsTime.label "time:">
-<!ENTITY pbSourceViewThemes.label "Theme">
-<!ENTITY compositionGoals.tooltip "Show or hide the Composition Goals status bar">
-<!ENTITY switchBetweenNormalAndCode.tooltip "Switch between Normal and Code views">
-<!ENTITY compositionGoals.label "Goals">
+<!ENTITY pbStatsChars.label "символов:">
+<!ENTITY formattingToolbar.label "Панель инструментов">
+<!ENTITY pbShowCompositionGoalsToolbarCmd.label "Строка состояния ограничений">
+<!ENTITY pbStatsTime.label "время:">
```

Comment:
CORRECTIONS NEEDED for line 410:
+<!ENTITY pbStatsTime.label "Время:">
Possible source error, since it's a label just like any other, and it should be capitalised normally.